PosterAgent/gen_pptx_code.py

```
@@ -25,7 +25,7 @@ def save_poster_code(output_file, utils_functions, presentation_object_name):
 '''
     return code
 
-def generate_panel_code(panel_dict, utils_functions, slide_object_name, visible=False, theme=None):
+def generate_panel_code(panel_dict, utils_functions, slide_object_name, visible=False, theme=None, lang: str = None):
```

Pylint (3.3.7)

[convention] Line too long (117/100)

(C0301)